src/components/WalletConnect/NanoContract/NewNanoContractTransactionRequest.js

```diff
@@ -0,0 +1,317 @@
+            <View style={styles.wrapper}>
+              {isTxInfoLoading() && (
+                <FeedbackContent
+                  title={t`Loading`}
+                  message={t`Loading transaction information.`}
+                  icon={<Spinner size={48} animating />}
+                  offmargin
+                  offcard
+                  offbackground
+                />
+              )}
+              {isTxInfoLoaded() && (
+                <View style={styles.content}>
+                  <DappContainer dapp={dapp} />
+                  <NanoContractExecInfo
+                    nc={ncToAccept}
+                    onSelectAddress={toggleSelectAddressModal}
+                  />
+                  <NanoContractActions
+                    ncActions={nc.actions}
+                    tokens={knownTokens}
+                    error={knownTokens.error}
+                  />
+                  <NanoContractMethodArgs
+                    blueprintId={nc.blueprintId}
+                    method={nc.method}
+                    ncArgs={nc.args}
+                  />
+
+                  {/* User actions */}
+                  <View style={styles.actionContainer}>
+                    <NewHathorButton
+                      title={t`Accept Transaction`}
+                      onPress={onAcceptTransaction}
+                    />
+                    <NewHathorButton
+                      title={t`Decline Transaction`}
+                      onPress={onDeclineTransaction}
+                      secondary
+                      danger
+                    />
+                  </View>
+                </View>
+              )}
+              {isTxProcessing() && (
+                <FeedbackContent
+                  title={t`Sending transaction`}
+                  message={t`Please wait.`}
+                  icon={<Spinner size={48} animating />}
+                  offmargin
+                  offcard
+                  offbackground
+                />
+              )}
+            </View>
+          </TouchableWithoutFeedback>
+        </ScrollView>
+      )}
+
+      <DeclineModal
+        show={showDeclineModal}
+        onDecline={onDeclineConfirmation}
+        onDismiss={onDismissDeclineModal}
+      />
+      <SelectAddressModal
+        address={ncAddress}
+        show={showSelectAddressModal}
+        onDismiss={toggleSelectAddressModal}
+        onSelectAddress={handleAddressSelection}
+      />
+      {isTxSuccessful() && (
+        <FeedbackModal
+          icon={(<Image source={checkIcon} style={styles.feedbackModalIcon} resizeMode='contain' />)}
+          text={t`Transaction successfully sent.`}
+          onDismiss={onFeedbackModalDismiss}
+          action={(<NewHathorButton discrete title={t`Ok, close`} onPress={onNavigateToDashboard} />)}
+        />
+      )}
+      {isTxFailed() && (
+        <FeedbackModal
+          icon={(<Image source={errorIcon} style={styles.feedbackModalIcon} resizeMode='contain' />)}
```

Comment:
`styles.feedbackModalIcon` is not defined

Reply:
Added.